client-js/app/elements/labrad-registry.html

```diff
@@ -25,51 +20,69 @@
       width: 100%;
     }
 
-    tbody tr:nth-child(1n).iron-selected {
-      background-color: #CCCCCC;
+    #outer {
+      flex-direction: column;
+      position: relative;
     }
-
-    .iron-selector {
+    #outer, #listing, #combinedList {
+      flex: 1;
+      display: flex;
       width: 100%;
-      display: block;
-      border: 1px solid #ccc;
-      border-top: none;
     }
-
-    table {
-      width: 100%;
-      border-collapse: collapse;
+    iron-list {
+      --iron-list-items-container: {
+        width: 100%;
+      };
     }
 
-    tbody {
-      display:table-header-group;
+    .row {
+      cursor: pointer;
+      display: flex;
+      flex-direction: row;
+      align-items: middle;
     }
-
-    td.dir.over {
-      background-color: #93d4d4;
+    .row:nth-child(1n).iron-selected {
+      background-color: #CCCCCC;
     }
-    tbody tr:nth-child(odd) {
+    .row:nth-child(odd) {
       background-color: #EEEEEE;
     }
-    tbody tr:nth-child(odd):hover {
+    .row:nth-child(odd):hover {
       background-color: #F6F699;
     }
-    tbody tr:nth-child(even):hover {
+    .row:nth-child(even):hover {
       background-color: #FFFFAA;
     }
-
-    .table-header {
-      @apply(--paper-font-body1);
-      color: var(--secondary-text-color);
-      *text-align: middle;
-      border-bottom: 1px solid;
-      border-bottom-color: #e5e5e5;
-      display:table-header-group;
-
+    .row .dir.over {
+      background-color: #93d4d4;
     }
-    .label-wide {
+    .row .dir {
+      white-space: nowrap;
+      -webkit-user-select: none;
+    }
+    .row .key {
+      width: 400px;
```

Comment:
This seems a little larger than we need for typical key names. I would suggest making a bit smaller and expressing in character size units, something like `16em` seems reasonable. People tend to keep registry windows small, half-screen or less, and we should leave as much room as possible for the values.